test/module/irohad/synchronizer/synchronizer_test.cpp

```diff
@@ -28,6 +31,17 @@ using ::testing::ByMove;
 using ::testing::DefaultValue;
 using ::testing::Return;
 
+/**
+ * Factory for generation mock mutable storages.
+ * This method provide technique,
```

Comment:
```suggestion
 * This method provides technique,
```

Reply:
I understand that this is not your code, but since we move it anyway, might as well correct it.